jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -185,7 +187,7 @@ pipeline {
                     String title = "[WIP] ${productName} update"
                     String body = "Created automatically by jenkins"
                     String[] defaultPRLabels = ["do-not-merge/work-in-progress"]
-                    if (params.dryRun) {
+                    if (params.dryRun && doPush) {
```

Comment:
This seemed to be causing me issues. It seems to be interpreting it as a function https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/fuse-online/job/discovery/11/console

Reply:
Hmm. Not sure why this change got added in this pr, I'll remove it.